Elmanager/Forms/ReplayViewer.Designer.cs

```diff
@@ -183,6 +187,20 @@ private void InitializeComponent()
             this.groundtouchesToolStripMenuItem.Size = new System.Drawing.Size(156, 22);
             this.groundtouchesToolStripMenuItem.Text = "Groundtouches";
             // 
+            // throttleOnToolStripMenuItem
+            // 
+            this.throttleOnToolStripMenuItem.CheckOnClick = true;
+            this.throttleOnToolStripMenuItem.Name = "throttleOnToolStripMenuItem";
+            this.throttleOnToolStripMenuItem.Size = new System.Drawing.Size(156, 22);
+            this.throttleOnToolStripMenuItem.Text = "Throttle on";

```

Comment:
I'd tell the user somewhere that gas times are just approximate times. This seems like a good place to do that, so here the text would be `Gas on (approx)`. Same for the off item.